### ISSUE TITLE
Pretrained Models and Torch

### ISSUE THREAD

**Issue by pansk** [NVIDIA/DIGITS#1352]
I wanted to move a torch model I trained on DIGITS on a remote machine to a local machine, and run some tests.

I encountered several problem (note: I updated and re-run these tests on the latest master version):

0) No matter how I upload a model, I cannot run inference directly on it, but should transform it into a normal Model, I assume by running at least 1 epoch (0 is invalid) with a very low LR

1) I uploaded the pretrained model downloading the "zip"and using "Upload Tar or Zip Archive" function. When I start a new training based on the pretrained network, and receive: "ParseError
1:1 : Message type "caffe.NetParameter" has no field named "require"."
Looks like it's interpreting the pretrained model as a caffe model.

2) I expanded the zip, gave a name, selected "torch" framework, selected correct image type, resize mode, width and height, uploaded weights and model definition (no labels). Start new training: fails (same problem)

3) Take the network as 2), create a new training, select custom network, click on "customize" beside the pretrained network, and then run the training: starts correctly

I think by default pretrained networks are interpreted as Caffe despite their actual content.

**Comment by gheinrich** [comment]
Thanks for the feedback - @IsaacYangSLA can you comment?
